src/pages/evm/wallet/SendPage.vue

```diff
@@ -278,9 +277,9 @@ export default defineComponent({
     <template v-slot:header>
         <div class="c-send-page__title-container">
             <div class="c-send-page__title"> {{ $t('evm_wallet.send') }}</div>
-            <div class="c-send-page__title-from">{{ $t('evm_wallet.from') }}</div>
+            <div class="c-send-page__title-from">{{ $t('global.from') }}</div>
             <UserInfo
-                class="c-send-page__title-address"
+                class="c-send-page__title-address c-user-info--lightweight"
```

Comment:
sort of has the same shadowing problem. this should be passed as a prop like 'lightweightText' or something, and set the class inside the userinfo component

Reply:
got it!